layers/descriptor_sets.cpp

```diff
@@ -1225,7 +1225,7 @@ bool CoreChecks::ValidateImageDescriptor(const char *caller, const DrawDispatchV
                     return LogError(set, vuids.descriptor_valid,
                                     "Descriptor set %s encountered the following validation error at %s time: Image layout "
                                     "specified "
-                                    "at vkUpdateDescriptorSet* or vkCmdPushDescriptorSet* time "
+                                    "at vkUpdateDescriptorSet* or vkCmdBindDescriptorSet* time "
```

Comment:
`vkCmdPushDescriptorSet*` can still trigger this call, no? Is there a reason it is removed?

Reply:
```suggestion
                                    "at vkCmdBindDescriptorSets time "
```

Looking at the VUID, It specifically says 

> Descriptors in each bound descriptor set, specified via vkCmdBindDescriptorSets

@Ryp I realize this has been sitting around a while (sorry), would you mind rebasing with the fix, if you would prefer me to apply the change instead I would also be willing to do that